internal/pkg/cli/pipeline_init.go

```diff
@@ -301,12 +303,21 @@ func (o *initPipelineOpts) RequiredActions() []string {
 }
 
 func (o *initPipelineOpts) askPipelineName() error {
+	promptOpts := []prompt.PromptConfig{
+		prompt.WithFinalMessage("Pipeline name:"),
+	}
+
+	// only show suggestion if [repo]-[branch] is a valid pipeline name
```

Comment:
super tiny nit: I think we try to keep the comment as a full sentence, hence we should capitalize it and end it with a period.
```suggestion
	// Only show suggestion if [repo]-[branch] is a valid pipeline name.
```